jaxley/modules/base.py

```diff
@@ -1044,6 +1045,32 @@ def make_trainable(
                 f"Number of newly added trainable parameters: {num_created_parameters}. Total number of trainable parameters: {self.base.num_trainable_params}"
             )
 
+    def write_trainables(self, trainable_params: List[Dict[str, jnp.ndarray]]):
```

Comment:
Cool! This only works for Module tho, i.e. doing `net.cell(0).write_trainables()` will only write to `self.base.nodes` if `self == self.base`. To be able to call this from a `View` and for these changes to take affect for the module you need to write to `self.base.nodes.loc[self._nodes_in_view]`, instead of `self.nodes`. See docstring of `View`. I think it might actually be nice if you could write only the trainables in View.

Reply:
doesn't work, see comments